tool/Makefile

```diff
@@ -0,0 +1,130 @@
+# SYNOPSIS:
+#
+#   make [all]  - makes everything.
+#   make TARGET - makes the given target.
+#   make clean  - removes all files generated by make.
+
+# Please tweak the following variable definitions as needed by your
+# project, except GTEST_HEADERS, which you can use in your own targets
+# but shouldn't modify.
+
+
+# Where to find user code.
+USER_DIR = ../src
+
+# Flags passed to the preprocessor.
+# Set Google Test's header directory as a system directory, such that
+# the compiler doesn't generate warnings in Google Test headers.
+CPPFLAGS += -DUNIT_TEST
+
+# Flags passed to the C++ compiler.
+CXXFLAGS += -g -Wall -Wextra -pthread
+
+all : gc_decode
+
+clean :
+	rm -f  *.o gc_decode
+
+
+# All the IR protocol object files.
+PROTOCOLS = ir_NEC.o ir_Sony.o ir_Samsung.o ir_JVC.o ir_RCMM.o ir_RC5_RC6.o \
+            ir_LG.o ir_Mitsubishi.o ir_Fujitsu.o ir_Sharp.o ir_Sanyo.o ir_Denon.o ir_Dish.o \
+            ir_Panasonic.o ir_Whynter.o ir_Coolix.o ir_Aiwa.o ir_Sherwood.o \
+            ir_Kelvinator.o ir_Daikin.o ir_Gree.o ir_Pronto.o ir_GlobalCache.o 
+# Common object files
+COMMON_OBJ = IRutils.o IRtimer.o IRsend.o IRrecv.o $(PROTOCOLS) 
+
+# Common dependencies
+COMMON_DEPS = $(USER_DIR)/IRrecv.h $(USER_DIR)/IRsend.h $(USER_DIR)/IRtimer.h \
+              $(USER_DIR)/IRutils.h $(USER_DIR)/IRremoteESP8266.h
+# Common test dependencies
+COMMON_TEST_DEPS = $(COMMON_DEPS) IRsend_test.h
```

Comment:
per the other comment, perhaps `../test/IRsend_test.h` instead? or what ever makes it use the original file.